.werft/jobs/build/installer/installer.ts

```diff
@@ -277,6 +277,12 @@ EOF`);
         exec(`yq w -i ${this.options.installerConfigPath} experimental.webapp.usage.defaultSpendingLimit.minForUsersOnStripe 1000`, { slice: slice })
         exec(`yq w -i ${this.options.installerConfigPath} experimental.webapp.usage.creditsPerMinuteByWorkspaceClass['default'] 0.1666666667`, { slice: slice })
         exec(`yq w -i ${this.options.installerConfigPath} experimental.webapp.usage.creditsPerMinuteByWorkspaceClass['gitpodio-internal-xl'] 0.3333333333`, { slice: slice })
+
+        // Configure Price IDs
+        exec(`yq w -i ${this.options.installerConfigPath} experimental.webapp.stripe.individualUsagePriceIds['EUR'] price_1LmYVxGadRXm50o3AiLq0Qmo`, { slice: slice })
```

Comment:
I see empty strings in the installer config when I ran `kubectl get configmap usage -o yaml`, but I expected these ones for preview?

Reply:
Ah, it's gonna be because changes to `.werft` don't get triggered by PRs. I need to manually trigger.

Reply:
I've [triggered the build](https://werft.gitpod-dev.com/job/gitpod-build-mp-usage-configure-price-ids.4). If you look at the preview env now, it does contain the config

Reply:
Thanks, looks good!